stock_move_backdating/models/stock_move.py

```diff
@@ -21,5 +21,10 @@ def action_done(self):
                 move.date = operation.operation_id.date
                 if move.quant_ids:
                     move.quant_ids.sudo().write({'in_date': move.date})

```

Comment:
I always have a difficulty here with this kind of modules. Not sure this is valid anymore in v8 but what if there is a Physical inventory/stock taking already approved for the product.
What would be the impact of the change in the date in the result of the stock taking?